django_social_share/templatetags/social_share.py

```diff
@@ -111,3 +112,21 @@ def post_to_gplus(context, obj_or_url=None, link_text='Post to Google+'):
     context = post_to_gplus_url(context, obj_or_url)
     context['link_text'] = link_text
     return context
+
+
+@register.simple_tag(takes_context=True)
+def post_to_mail_url(context, text, subject, obj_or_url=None):
```

Comment:
Whatever we do, values will always be just defaults that the user can change in their email client before sending the email. If developers don't want to specify a default for the subject, they can easily pass the empty string.